src/api/procedures/__tests__/reserveTicker.ts

```diff
@@ -28,24 +29,36 @@ jest.mock(
 describe('reserveTicker procedure', () => {
   let mockContext: Mocked<Context>;
   let stringToTickerStub: sinon.SinonStub<[string, Context], Ticker>;
+  let posRatioToBigNumberStub: sinon.SinonStub<[PosRatio], BigNumber>;
+  let balanceToBigNumberStub: sinon.SinonStub<[Balance], BigNumber>;
   let ticker: string;
   let rawTicker: Ticker;
   let args: ReserveTickerParams;
   let fee: number;
   let reservation: PostTransactionValue<TickerReservation>;
+  let rawPosRatio: PosRatio;
+  let rawFee: Balance;
+  let numerator: number;
+  let denominator: number;
 
   beforeAll(() => {
     polkadotMockUtils.initMocks({ contextOptions: { balance: new BigNumber(500) } });
     procedureMockUtils.initMocks();
     entityMockUtils.initMocks({ identityOptions: { did: 'someOtherDid' } });
     stringToTickerStub = sinon.stub(utilsModule, 'stringToTicker');
+    posRatioToBigNumberStub = sinon.stub(utilsModule, 'posRatioToBigNumber');
+    balanceToBigNumberStub = sinon.stub(utilsModule, 'balanceToBigNumber');
     ticker = 'someTicker';
     rawTicker = polkadotMockUtils.createMockTicker(ticker);
     args = {
       ticker,
     };
     fee = 250;
     reservation = ('reservation' as unknown) as PostTransactionValue<TickerReservation>;
+    rawPosRatio = polkadotMockUtils.createMockPostRatio(numerator, denominator);
+    rawFee = polkadotMockUtils.createMockBalance(fee);
+    numerator = 1;
```

Comment:
You should test with values different than 1 to make sure that the division and multiplication are actually being done